opencensus/exporters/trace/zipkin/zipkin_exporter.h

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef OPENCENSUS_EXPORTERS_TRACE_ZIPKIN_ZIPKIN_EXPORTER_H_
+#define OPENCENSUS_EXPORTERS_TRACE_ZIPKIN_ZIPKIN_EXPORTER_H_
+
+#include <memory>
+#include <string>
+
+#include "absl/memory/memory.h"
```

Comment:
I don't think memory or absl/memory are needed anymore. Ditto span_exporter and span

Reply:
span.h -> removed.
absl/memory/memory.h -> needed for WrapUnique, moved to .cc file
span_exporter.h -> needed for handler and SpanData definitions, moved to .cc file